runtime/bajun/src/lib.rs

```diff
@@ -558,7 +561,7 @@ impl orml_vesting::Config for Runtime {
 	type VestedTransferOrigin = EnsureSigned<AccountId>;
 	type WeightInfo = weights::orml_vesting::WeightInfo<Runtime>;
 	type MaxVestingSchedules = frame_support::traits::ConstU32<100>;
-	type BlockNumberProvider = RelaychainDataProvider<Runtime>;
+	type BlockNumberProvider = System;
```

Comment:
Can we maybe have a comment here of the rationale? Just that we never forget. :)